Proxy/internal/database/model/model.go

```diff
@@ -7,15 +7,15 @@ import (
 )
 
 type Cluster struct {
-	bun.BaseModel `bun:"table:Clusters"`
+	bun.BaseModel `bun:"table:clusters"`
 	ID            int       `bun:"id,autoincrement"`
 	ClusterID     int       `bun:"cluster_id"`
 	Timestamp     time.Time `bun:"timestamp,notnull"`
 	Name          string    `bun:"name"`
 }
 
 type Service struct {
-	bun.BaseModel     `bun:"Services"`
+	bun.BaseModel     `bun:"services"`
```

Comment:
Why do we use `bun:"table:clusters"` sometimes and `bun:"services"` in other times?